src/styles/_card.scss

```diff
@@ -0,0 +1,97 @@
+@import './variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: $font-family-main;
+  box-sizing: border-box;
+  width: 199px;
+  padding: $padding-large $padding-medium $padding-medium;
+  border: 1px solid $border-color;
+  background-color: $secondary-color;
+
+  &__product-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    top: 33px;
```

Comment:
The `top` and `left` properties are not relevant for block-level elements. You could use `margin` or `padding` instead to position your elements.